nb2workflow/nbadapter.py

```diff
@@ -813,9 +813,16 @@ def download_file(self, file_url, tmpdir):
     def handle_url_params(self, parameters, tmpdir, context={}):
         adapted_parameters = copy.deepcopy(parameters)
         exceptions = []
+        ontology_available = False
+        if len(ontology.g) > 0:
```

Comment:
This could be not a proper check.

I imagine changes in oda_api.ontology_helper to allow to read parameters' `extra_ttl` even if there's no ontology loaded. Then the graph will be populated at this moment. 

The way that comes to mind is to have some instance method as a flag to this mode of operation. (You may invent it here and add an issue to oda_api to not forget. Or you may override what's needed in the inherited ontology class here).

Reply:
what is to be checked in order to verify whether or not the ontology is available then ?

Reply:
It can be determined when creating an instance, I think. Do one pass ontology path there and is it available and reachable